packages/app/obojobo-document-engine/src/scripts/viewer/util/recommendation-util.js

```diff
@@ -0,0 +1,99 @@
+import NavStore from '../stores/nav-store'
+
+const RecommendationUtil = {
```

Comment:
The idea behind the Util classes is that they either get data out of the state objects you give them or they simply emit the correct event for you. The ones in here that mutate state, like boundConfidence, onOpenQuestion and such, should move to recommendation-store (they can be "private" methods in recommendation-store)